rpc/cloudagent/network/logicalnetwork/moc_cloudagent_logicalnetwork.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/network";
+package moc.cloudagent.network;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+import "moc_common_networkcommon.proto";
+
+message LogicalNetworkRequest {
+	repeated LogicalNetwork LogicalNetworks = 1;
+	Operation OperationType = 2;
+}
+
+message LogicalNetworkResponse {
+	repeated LogicalNetwork LogicalNetworks = 1;
+	google.protobuf.BoolValue Result = 2;
+	string  Error = 3;
+}
+
+message LogicalNetwork {
+	string name = 1;
+	string id = 2;
+	repeated LogicalSubnet subnets = 3;
+	bool networkVirtualizationEnabled = 4;
+	string nodefqdn = 5;
```

Comment:
Why do we need nodeFqdn

Reply:
That got copied over from VirtualNetwork definition. Seems that is not being used. I can remove it.

Reply:
Resolved - Removed nodefqdn.